x-pack/plugins/observability_solution/entity_manager/server/routes/enablement/check.ts

```diff
@@ -44,9 +43,8 @@ export function checkEntityDiscoveryEnabledRoute<T extends RequestHandlerContext
           return res.ok({ body: { enabled: false, reason: ERROR_API_KEY_NOT_VALID } });
         }
 
-        const fakeRequest = getFakeKibanaRequest({ id: apiKey.id, api_key: apiKey.apiKey });
-        const soClient = server.core.savedObjects.getScopedClient(fakeRequest);
-        const esClient = server.core.elasticsearch.client.asScoped(fakeRequest).asCurrentUser;
+        const esClient = (await context.core).elasticsearch.client.asCurrentUser;
```

Comment:
this change ensures that only users with correct permissions can check enablement status? wouldn't we want anyone to be able to call this check endpoint?

Reply:
Yes, not sure about this. can revert to using the api key, it's a read only api so shouldn't cause much harm to have anyone able to read the response